src/components/PaginationFooter/PaginationFooter.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useEffect } from 'react';
+import { Select, Stack, Text, ChakraProvider } from '@chakra-ui/react';
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'ChakraProvider' is defined but never used.